conbench/entities/run.py

```diff
@@ -150,3 +205,44 @@ def commit_hardware_run_map():
         results[commit]["hardware"][run.hardware.name].append((run_value, run.id))
 
     return results
+
+
+class GitHubCreate(marshmallow.Schema):
+    commit = marshmallow.fields.String(required=True)
+    repository = marshmallow.fields.String(required=True)
+
+
+class _RunFacadeSchemaCreate(marshmallow.Schema):
+    id = marshmallow.fields.String(required=True)
+    name = marshmallow.fields.String(required=False)
+    reason = marshmallow.fields.String(required=False)
+    finished_timestamp = marshmallow.fields.DateTime(required=False)
+    info = marshmallow.fields.Dict(required=False)
+    error_info = marshmallow.fields.Dict(required=False)
+    error_type = marshmallow.fields.String(required=False)
```

Comment:
What's the intention for how `error_type` is to be used separately from `error_info`? I'm assuming it's supposed to be a low-cardinality value about where it failed, and not an error class (which would go in `error_info`)? Can we document this somewhere? Ideally in descriptions in the swagger, but I'm not sure how hard that is with the way things are set up

Reply:
Documented all new fields in swagger:
<img width="1035" alt="Screen Shot 2022-10-21 at 12 07 50 PM" src="https://user-images.githubusercontent.com/5522472/197271908-0492aef5-cb8d-4f2c-8265-e3ef044d8ef0.png">

Reply:
<img width="798" alt="Screen Shot 2022-10-21 at 12 14 00 PM" src="https://user-images.githubusercontent.com/5522472/197272184-457b56da-4b0a-4ee6-89a0-d4b82cb8c8f2.png">